ush/generate_FV3LAM_wflow.sh

```diff
@@ -423,6 +424,7 @@ settings="\
   'do_spinup': ${DO_SPINUP}
   'do_nldn_lght': ${DO_NLDN_LGHT}
   'regional_ensemble_option': ${regional_ensemble_option}
+  'ensctrl_stmp': ${ENSCTRL_STMP}
```

Comment:
This is useful in the rocoto template so we can wait on output from the control member before starting the graphics.